Makefile

```diff
@@ -83,13 +83,10 @@ goword:tools/bin/goword
 gosec:tools/bin/gosec
 	tools/bin/gosec $$($(PACKAGE_DIRECTORIES))
 
-check-static:tools/bin/gometalinter tools/bin/misspell tools/bin/ineffassign
-	@ # TODO: enable megacheck.
-	@ # TODO: gometalinter has been DEPRECATED.
-	@ # https://github.com/alecthomas/gometalinter/issues/590
-	tools/bin/gometalinter --disable-all --deadline 120s \
-	  --enable misspell \
-	  --enable ineffassign \
+check-static: tools/bin/golangci-lint
+	tools/bin/golangci-lint run -v --disable-all --deadline=3m \
```

Comment:
why changed to 3m ?

Reply:
Indeed, golangci-lint is faster than gometalinter. 
But it cannot support like `--deadline 120s` , therefore I changed it to `--deadline=3m` .
The reason for 3m rather than 2m , beacuse I think we should give more time for the new tool. 
And changed it back when it stable enough.